nx-X11/programs/Xserver/hw/nxagent/Args.c

```diff
@@ -1050,21 +935,19 @@ int ddxProcessArgument(int argc, char *argv[], int i)
 
   if (!strcmp(argv[i], "-verbose"))
   {
-    nxagentVerbose = 1;
-
+    nxagentVerbose = True;
     return 1;
   }
 
   if (!strcmp(argv[i], "-keystrokefile"))
   {
     if (i + 1 < argc)
     {
-      if (NULL != (nxagentKeystrokeFile = strdup(argv[i + 1])))
+      if ((nxagentKeystrokeFile = strdup(argv[i + 1])) == NULL)
```

Comment:
I'd keep that as `NULL == ...`

Reply:
I fully get you point here, but I want to keep the code uniform within a file. And the rest of the code has the constant at the right side. The Xorg code is also using this technique.

Reply:
Welll... okay.

Personally, I'll be keeping this style for new submissions, particularly since it's just a cosmetic issue and provides a potential benefit.

I totally respect your decision, though. It's fine.